client/app/index.jsx

```diff
@@ -3,7 +3,27 @@ import {render} from 'react-dom';
 
 class App extends React.Component {
   render () {
-    return <p>HELLO ___ REACT!</p>;
+    return (
+      <div className="parent">
```

Comment:
Probably don't need all theses classes right now on all these divs unless you are going to use them for something right now or soon.

Also, two style convention things for class names

1. No need to put the tag type in the class name. Eg, `maindiv` should just be `main`
2. Use "kebab case" like `main-div` instead of run-on like `maindiv` or camelCase like `mainDiv`

Suggestions
1. Remove classes that aren't being used right now